etc/make-macos-x64-build.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+package_version=$(cat package.json | jq -r '.version')
```

Comment:
```suggestion
package_version=$(node -p "require('./package.json').version")
```

Very minor but it is nice to not add a need for `jq` when node is already going to be around for anyone working on/with this repo. (matching `etc/install-zstd.sh`)